Patches/Rabbie The Moonrabbit/ThingDefs_Misc/Weapons_Guns.xml

```diff
@@ -76,6 +76,124 @@
+					<li Class="CombatExtended.ToolCE">
+					  <label>neck</label>
+					  <capacities>
+						<li>Poke</li>
+					  </capacities>
+					  <power>2</power>
+					  <cooldownTime>1.54</cooldownTime>
+					  <armorPenetrationBlunt>0.555</armorPenetrationBlunt>
+					</li>
+				  </tools>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Shell_HighExplosive"]/verbs/li[verbClass="Verb_ShootOneUse"]</xpath>
+				<value>
+				<li Class="CombatExtended.VerbPropertiesCE">
+					<verbClass>CombatExtended.Verb_ShootCEOneUse</verbClass>
+					<hasStandardCommand>true</hasStandardCommand>
+					<defaultProjectile>Throw_81mmMortarShell_HE</defaultProjectile>
+					<warmupTime>2.5</warmupTime>
+					<range>7</range>
+					<ignorePartialLoSBlocker>true</ignorePartialLoSBlocker>
+					<targetParams>
+						<canTargetLocations>true</canTargetLocations>
+					</targetParams>
+				</li>
+				</value>
+			</li>
+			
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/RecipeDef[defName="MakeShell_HighExplosive"]</xpath>
+				<value>
+					<recipeUsers>
+						<li>TableMachining</li>
+					</recipeUsers>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs</xpath>
+				<value>
+				  <ThingDef ParentName="Base81mmMortarShell">
+					<defName>Throw_81mmMortarShell_HE</defName>
+					<label>81mm mortar shell (HE)</label>
+					<graphicData>
+					  <texPath>Things/Projectile/Mortar/HE</texPath>
+					  <graphicClass>Graphic_Single</graphicClass>
+					</graphicData>
+					<projectile Class="CombatExtended.ProjectilePropertiesCE">
+					  <speed>5</speed>
+					  <damageDef>Bomb</damageDef>
+					  <damageAmountBase>156</damageAmountBase>
+					  <armorPenetrationSharp>0</armorPenetrationSharp>
+					  <armorPenetrationBlunt>0</armorPenetrationBlunt>
+					  <explosionRadius>2.5</explosionRadius>
+					  <soundHitThickRoof>Artillery_HitThickRoof</soundHitThickRoof>
+					  <soundImpactAnticipate/>
+					  <soundAmbient/>
+					  <flyOverhead>false</flyOverhead>
+					  <dropsCasings>false</dropsCasings>
+					  <soundExplode>MortarBomb_Explode</soundExplode>
+					  <applyDamageToExplosionCellsNeighbors>true</applyDamageToExplosionCellsNeighbors>
+					  <ai_IsIncendiary>true</ai_IsIncendiary>
+					  <gravityFactor>2</gravityFactor>
+					</projectile>
+					<comps>
+					  <li Class="CombatExtended.CompProperties_Fragments">
+						<fragments>
+							<Fragment_Large>16</Fragment_Large>
+							<Fragment_Small>100</Fragment_Small>
+						</fragments>
+					  </li>
+					</comps>
+				  </ThingDef>
+				</value>
+			</li>
+			
+			<!--Add large machining table for VFE:Production -->
+			
+			<li Class="PatchOperationFindMod">
+				<mods>
+				  <li>Vanilla Furniture Expanded - Production</li>
+				</mods>
+				<match Class="PatchOperationAdd">
+					<xpath>Defs/RecipeDef[defName="MakeShell_HighExplosive"]/recipeUsers</xpath>
+					<value>
+						<li>VFE_TableMachiningLarge</li>
```

Comment:
I'm a bit confused by this. Is it related to the Moonrabbit patch? Should it go into the VFE - Production patch folder?

Reply:
It is only needed if Rabbie _and_ VFE: Production are both loaded. I figured it was easier to group it with the rest of the changes here rather than put it in VFE:Production.